packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -194,6 +197,7 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
                   ({ status }) => {
                     if (status === 201) {
                       mutate();
```

Comment:
Nit: It'd be nice to rename this `mutateVisits` or something similar to disambiguate it from the queue entry mutator function.

Reply:
Done